src/pcm.rs

```diff
@@ -178,6 +178,23 @@ impl PCM {
         acheck!(snd_pcm_status(self.0, z.ptr())).map(|_| z)
     }
 
+    pub fn status_with_audio_htstamp_config(
+        &self,
+        type_requested: AudioTstampType,
+        report_delay: bool,
+    ) -> Result<Status> {
+        let z = Status::new();
+        let mut cfg = alsa::snd_pcm_audio_tstamp_config_t {
+            _bitfield_1: alsa::snd_pcm_audio_tstamp_config_t::new_bitfield_1(
+                type_requested as libc::c_uint,
+                libc::c_uint::from(report_delay),
+            ),
+            __bindgen_padding_0: [0; 3],
+        };
+        unsafe { alsa::snd_pcm_status_set_audio_htstamp_config(z.ptr(), &mut cfg) };
```

Comment:
I would prefer if we avoid the names `_bitfield_1` and `__bindgen_padding_0` when possible. Bindgen has a tendency to change them occasionally and it's good to be resilient (also IMO the code will be more readable).
I e, 

    let mut cfg = mem::zeroed();
    cfg.set_requested_type(...);
    cfg.set_report_delay(...);

Reply:
Yes, that's better. I haven't worked much with bindgen. Thanks for the suggestion.